### DISCUSSION TITLE
Two ideas!

### DISCUSSION THREAD

**Discussion by Madisongs** [BedrockTweaks/Files#494] [Ideas]
I’ve already made textures for this first one in my personal pack, but it always bugged me how the music discs were 15 wide instead of 16. So, 16 wide discs!
Another suggestion is bowl consistency. Soup, stew, and regular bowls aren’t the same height despite being the same shape. Which is weird! Those are my suggestions :]

**Comment by SquatchHunter** [comment]
i'll allow it
however it wont be high priority or considered till after addons release
